### ISSUE TITLE
breaking updates on elixir 1.17

### ISSUE THREAD

**Issue by DohanKim** [bonfire-networks/iconify_ex#10]
when I compiled Iconify with Elixir 1.17, I got these errors.

```
     error: undefined function .../0 (expected Iconify to define such a function or for it to be imported, but none are available)
     │
 827 │         |> Phx.Live.Favicon.set_dynamic(socket, "svg", ...)
     │                                                        ^^^
     │
     └─ lib/iconify.ex:827:56: Iconify.maybe_set_favicon_emoji/2

     error: undefined function .../0 (expected Iconify to define such a function or for it to be imported, but none are available)
     │
 851 │     |> Phx.Live.Favicon.set_dynamic(socket, "svg", ...)
     │                                                    ^^^
     │
     └─ lib/iconify.ex:851:52: Iconify.do_set_favicon_text/2

     error: undefined function .../0 (expected Iconify to define such a function or for it to be imported, but none are available)
     │
 858 │     |> Phx.Live.Favicon.set_dynamic(socket, "svg", ...)
     │                                                    ^^^
     │
     └─ lib/iconify.ex:858:52: Iconify.do_set_favicon_iconify/2


== Compilation error in file lib/iconify.ex ==
** (CompileError) lib/iconify.ex: cannot compile module Iconify (errors have been logged)
```

**Comment by mayel** [comment]
thanks for the report! I just found the same myself, seems to be an issue with our https://github.com/bonfire-networks/arrows library, will address it in https://github.com/bonfire-networks/bonfire-app/issues/925

**Comment by mayel** [comment]
Possibly related to this in Elixir 1.17 changelog: `[Kernel] Resolve inconsistencies of how .. and ... are handled at the AST level`

**Comment by mayel** [comment]
Fixed in arrows 0.2 and iconify_ex 0.4.5
